extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -404,12 +404,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 			notebookPath = uri.fsPath;
 		}
 
-		if (shouldReveal || this._bookViewer?.visible) {
+		if (this._bookViewer?.visible) {
 			bookItem = notebookPath ? await this.findAndExpandParentNode(notebookPath) : undefined;
-			// Select + focus item in viewlet if books viewlet is already open, or if we pass in variable
 			if (bookItem?.contextValue && bookItem.contextValue !== 'pinnedNotebook') {
-				// Note: 3 is the maximum number of levels that the vscode APIs let you expand to
-				await this._bookViewer.reveal(bookItem, { select: true, focus: true, expand: true });
+				if (shouldReveal) {
+					// Select + focus item in viewlet if books viewlet is already open, or if we pass in variable
+					// Note: 3 is the maximum number of levels that the vscode APIs let you expand to
+					await this._bookViewer.reveal(bookItem, { select: true, focus: true, expand: true });
+				} else {
+					// Do not focus on item if element is already visible in the Tree View.
```

Comment:
This comment doesn't seem accurate - the logic here isn't determining whether the item is "visible". That should always be true after the call to findAndExpandParentNode. 

You're just changing the logic such that shouldReveal controls whether the item is brought into view instead. Which are we sure we want to change that behavior? 

What about adding another parameter that controls whether we focus the item instead? So shouldReveal would control whether it forced it to expand the parent node, while this other one controlled whether it was focused or not.